pkg/image/apis/image/docker10/conversion.go

```diff
@@ -0,0 +1,24 @@
+package docker10
+
+// Convert_DockerV1CompatibilityImage_to_DockerImageConfig takes a Docker registry digest
+// (schema 2.1) and converts it to the external API version of Image.
+func Convert_DockerV1CompatibilityImage_to_DockerImageConfig(in *DockerV1CompatibilityImage, out *DockerImageConfig) error {
+	*out = DockerImageConfig{
+		ID:              in.ID,
+		Parent:          in.Parent,
+		Comment:         in.Comment,
+		Created:         in.Created,
+		Container:       in.Container,
+		DockerVersion:   in.DockerVersion,
+		Author:          in.Author,
+		Architecture:    in.Architecture,
+		Size:            in.Size,
+		OS:              "linux",
```

Comment:
i feel like this is going to bite us as soon as we try to do multiarch...  do we not have the image's architecture info anywhere?  should this at least be based on runtime.GOARCH like we do on manifestlist conversion?